Libraries/src/Amazon.Lambda.RuntimeSupport/Amazon.Lambda.RuntimeSupport.csproj

```diff
@@ -18,6 +18,12 @@
   <PropertyGroup Condition=" '$(ExecutableOutputType)'=='true' ">
     <OutputType>Exe</OutputType>
   </PropertyGroup>
+	
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net8.0'">
+    <WarningsAsErrors>IL2026,IL2067,IL2075</WarningsAsErrors>
+    <IsTrimmable>true</IsTrimmable>
+	<EnableTrimAnalyzer>true</EnableTrimAnalyzer>
```

Comment:
Why does RuntimeSupport need `EnableTrimAnalyzer` but the rest don't?

Reply:
Added `EnableTrimAnalyzer` to the other projects. I added this property so we can see future breaks to trimming at compile time for these libraries.